From 8e0965646e6e00945d866c1131a261f1c4cf4fca Mon Sep 17 00:00:00 2001
From: Mitchell Freiderich <MrCowKing@users.noreply.github.com>
Date: Mon, 18 Jul 2022 01:03:44 -0500
Subject: [PATCH] hwdb: Add Acer Aspire A317-33 (#24050)

Enables the power button on the Acer Aspire 3 A317-33

(cherry picked from commit 0216462f0836596fb0b2bf7b819bfd1d5b45b951)

Related: #2087778
---
 hwdb.d/60-keyboard.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 51c631929e..095bb34cd8 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -154,6 +154,10 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspire*8930:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspire*7750G:*
  KEYBOARD_KEY_e0=!pageup
 
+# Acer Aspire 3 A317-33
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspireA317-33:*
+ KEYBOARD_KEY_55=power
+
 # Acer Aspire One AO532h
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAO532h:*
  KEYBOARD_KEY_84=bluetooth
